app/client/ui/WelcomeQuestions.ts

```diff
@@ -54,21 +57,21 @@ export function showWelcomeQuestions(userPrefsObs: Observable<UserPrefs>) {
 }
 
 const choices: Array<{icon: IconName, color: string, text: string}> = [
```

Comment:
```suggestion
const choices: Array<{icon: IconName, color: string, textKey: string}> = [
```

See comment below for context.